lib/payloadAction.ts

```diff
@@ -136,6 +136,17 @@ export const handlePayloadActions = async (
 					await setIsQueueWindowActive(read, persistence, rid, false);
 					await setQueuedMessage(read, persistence, rid, '');
 					console.debug({ rid }, 'Queue Window dropped');
+				} else if (actionName === ActionIds.FEEDBACK_SURVEY) {
+					if (!params) {
+						return;
+					}
+					const customFields = {} as IRoomCustomField;
+					if (params.value === true) {
+						customFields.feedbackSurvey = params.time;
+					} else {
+						customFields.feedbackSurvey = null;
+					}
+					await updateRoomCustomFields(rid, customFields, read, modify);
```

Comment:
@bhardwajaditya can we add a log point here when we are setting the customField for the feedbackSurvey We'll need to know if we are setting True or False, and include the RoomId please. Thanks.

Reply:
@bhardwajaditya I don't think I'm seeing the logpoints from this PR?  I do see the triggering feedback survey log point from the server.  Will console.log from the DF.app show up in RC logging?

Reply:
After deploying to our dev and preprod servers I am seeing the log points.  No idea what's going on with my personal dev server.... this PR looks good!